### PR TITLE
MINOR FIX : Teacher Service Exception

### PR DESCRIPTION
**Pull request by tharanisantosh**

### CODE REVIEW

File: OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.Teachers.Exceptions
+{
+    public class FailedTeacherServiceException : Xeption
+    {
+        public FailedTeacherServiceException(Exception innerException)
+            : base(message: "Failed teacher service error occured. ",innerException)
+        {
+
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs:11]: @tharanisantosh remove that extra empty line do CTRL + K + D

File: OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs:1]: Needs copyrights here.

Comment (tharanisantosh) [reply OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs]: Hi Hassan, Added Copyrights. Comment Resolved. Thanks.

File: OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs (hunk 1 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api/Models/Teachers/Exceptions/FailedTeacherServiceException.cs]: Removed extra line. Comment resolved.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**